internal/jobs/transform_test.go

```diff
@@ -204,26 +204,3 @@ var _ = Describe("Calling ToString from a transform", func() {
 		Expect(ret).To(Equal("&{map[] map[field1:hello] ns1:1 1 2 false}"))
 	})
 })
-
-var _ = Describe("Test that httpTransform picks up timeout from config", func() {
```

Comment:
timeout is still relevant? why remove

Reply:
There was a bunch of other scaffolding that was needed that wasnt quick to do so traded that off versus value of the test.